imsim/photon_ops.py

```diff
@@ -200,7 +200,7 @@ def applyTo(self, photon_array, local_wcs=None, rng=None):
             geometry=LSST_SPIDER_GEOMETRY,
             distribution=self.diffraction_rng(rng),
         )
-        photon_array.x, photon_array.y = xy_to_v.inverse(v)
+        photon_array.x[()], photon_array.y[()] = xy_to_v.inverse(v)
```

Comment:
Why this change?

Reply:
Indeed, I think, here we dont gain anything by this change.
Before: rhs is a new numpy.ndarray which then will replace lhs which should be garbage collected.
After: rhs is again a new numpy.ndarray which then will be copied to lhs and garbage collected afterwards.

Will remove it again

Reply:
Actually, the setter for x and y already write into the existing array.  So in both cases, the temporary returned by xy_to_v.inverse is the one that will be garbage collected.

Reply:
OK. I removed this commit.